docs/release-notes/cloud/cloud-2024-01-12-gh-repository-ssh-keys-discontinuation.md

```diff
@@ -0,0 +1,83 @@
+
+On the other hand, the usage of installation access tokens no longer requires [read and write repository permissions for Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration). The Codacy GitHub App still requests this permission during this first stage, while using repository SSH keys as a fallback mechanism. At a later stage, Codacy will [remove the permission from the GitHub App](#timeline).
+
+## What do you need to do if you have a GitHub organization? {: id="to-do"}
+
+Make sure an organization owner [approves the updated permissions for the Codacy GitHub App](https://docs.github.com/en/apps/using-github-apps/reviewing-and-modifying-installed-github-apps) on your GitHub organization, if not done yet.
+
+Codacy has been requesting repository read permissions for **Contents** since September 2023. Organization owners should have received a GitHub notification to review a request for this additional permission:
+
+![Codacy GitHub App updated permissions request](../images/2023-12-DD-gh-updated-permissions.png)<!--TODO PLUTO-802 Rename image-->
+
+If you have any questions or need help, please contact <mailto:support@codacy.com>.
+
+## Removal of repository permissions for Administration and SSH keys {: id="timeline"}
+
+To ensure the conditions to use installation access tokens on GitHub organizations are met before the removal of repository permissions for Administration and SSH keys, Codacy will execute a phased rollout according to the timeline below:
+
+<table>
+  <thead>
+    <th>Date</th>
+    <th>Event</th>
+    <th>What to expect</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>January 12, 2024</td>
+      <td>Codacy started using installation access tokens to clone and integrate with your repositories</td>
+      <td>
+        <p>From this day on, repository SSH keys are used exclusively as a fallback mechanism when the Contents permission is missing.</p>
+        <p>If not done yet, make sure an organization owner <a href="#to-do">approves Codacy GitHub App updated permissions</a> on your GitHub organization.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>February 12, 2024</td>
+      <td>Brownout of repository permissions for Administration</td>
+      <td>
+        <p>On this day, Codacy won't use any fallback mechanism if the Contents permission is missing.</p>
+        <p>This will help you confirm that the updated permissions for the Codacy GitHub App were already approved for your organization.<br/>If on this date your Codacy quality analysis fails or you have problems adding new repositories to Codacy, make sure an organization owner <a href="#to-do">approves the updated permissions for the Codacy GitHub App</a> on your GitHub organization.</p>
+      </td>
+    </tr>
+    <tr>
+      <td>February 19, 2024</td>
```

Comment:
I've questioned if we want to comunicate this date and leave the 12th as THE date for the clients. - waiting to hear from @jorgebraz

Reply:
Let's keep it, as talked.